src/index.tsx

```diff
@@ -18,7 +18,7 @@ export const PINWHEEL_MESSAGE_TYPES = {
   PINWHEEL_EVENT: 'PINWHEEL_EVENT',
 };
 
-const LINK_PAGE_URL = 'https://cdn.getpinwheel.com/link-v3.1.0.html';
+const LINK_PAGE_URL = 'https://cdn.getpinwheel.com/link-v3.2.0.html';
```

Comment:
we bumped to 2.2 right?

Reply:
Yup this was the piece I mentioned I have to update here